week_6/src/main/scala/org/epfl/Future_sequence_fail_fast.scala

```diff
@@ -0,0 +1,67 @@
+package org.epfl.future.sequence.fail.fast
```

Comment:
I think it should be in `part9`

Reply:
This example is interesting, but I think it’s a bug in the Scala library. See https://github.com/scala/bug/issues/8994

I’m not sure we want to explain these things to the students?

Reply:
Interesting...
I wasn't aware it was flagged as a bug. On the other hand, it's been open since almost 6 years!
We could still leave it in as an example without actually showing it to the students.

Reply:
OK.

Reply:
> I think it should be in `part9`

Moved it to that section